src/main/java/com/actiontech/dble/meta/ReloadLogHelper.java

```diff
@@ -28,96 +28,64 @@ public ReloadLogHelper(boolean isReload) {
-        }
+    public void warn(String message) {
+        LOGGER.warn(getStage() + message);
     }
 
-    public void info(String message) {
-        LOGGER.info(getStage() + message);
+    public void warn(String message, Throwable var2) {
+        LOGGER.warn(getStage() + message, var2);
     }
 
-    public static void debug(String message, Logger logger) {
-        if (!logger.isDebugEnabled()) {
-            return;
-        }
-        if (ReloadManager.getReloadInstance().getStatus() != null) {
-            logger.debug(ReloadManager.getReloadInstance().getStatus().getLogStage() + message);
-        } else {
-            logger.debug(message);
-        }
+    private String getStage() {
+        return reload == null ? "" : reload.getLogStage();
     }
 
-    public static void debug(String message, Logger logger, Object val) {
-        if (!logger.isDebugEnabled()) {
-            return;
-        }
-        if (ReloadManager.getReloadInstance().getStatus() != null) {
-            logger.debug(ReloadManager.getReloadInstance().getStatus().getLogStage() + message, val);
-        } else {
-            logger.debug(message, val);
-        }
+    public boolean isReload() {
+        return isReload;
     }
 
-    public static void debug(String message, Logger logger, Object... val) {
-        if (!logger.isDebugEnabled()) {
-            return;
-        }
-        if (ReloadManager.getReloadInstance().getStatus() != null) {
-            logger.debug(ReloadManager.getReloadInstance().getStatus().getLogStage() + message, val);
-        } else {
-            logger.debug(message, val);
-        }
-    }
 
-    public static void warn(String message, Logger logger) {
-        if (ReloadManager.getReloadInstance().getStatus() != null) {
-            logger.info(ReloadManager.getReloadInstance().getStatus().getLogStage() + message);
-        } else {
-            logger.info(message);
-        }
+    // ========= static method
+    public static String getLogStage() {
+        return ReloadManager.getReloadInstance().getStatus() != null ? ReloadManager.getReloadInstance().getStatus().getLogStage() : "";
     }
 
-    public static void warn(String message, Throwable var2, Logger logger) {
-        if (ReloadManager.getReloadInstance().getStatus() != null) {
-            logger.info(ReloadManager.getReloadInstance().getStatus().getLogStage() + message, var2);
-        } else {
-            logger.info(message, var2);
-        }
+    public static void debug(String message, Object... val) {
+        if (!LOGGER.isDebugEnabled()) return;
+        LOGGER.debug(getLogStage() + message, val);
     }
 
-    public void warn(String message) {
-        LOGGER.warn(getStage() + message);
+    public static void info2(String message) {
+        LOGGER.info(getLogStage() + message);
     }
 
-    public void warn(String message, Throwable var2) {
-        LOGGER.warn(getStage() + message, var2);
+    public static void info3(String message) {
```

Comment:
info2 and info3 need straightforward method name

Reply:
get